packages/tenant-process/src/services/tenantService.ts

```diff
@@ -430,42 +429,32 @@ export function tenantServiceBuilder(
       {
         tenantId,
         attributeId,
-        organizationId,
-        correlationId,
       }: {
         tenantId: TenantId;
         attributeId: AttributeId;
-        organizationId: TenantId;
-        correlationId: string;
       },
-      logger: Logger
+      { authData, correlationId, logger }: WithLogger<AppContext>
     ): Promise<void> {
       logger.info(
         `Revoke certified attribute ${attributeId} to tenantId ${tenantId}`
       );
       const requesterTenant = await retrieveTenant(
-        organizationId,
+        authData.organizationId,
         readModelService
       );
 
-      const certifierId = requesterTenant.data.features.find(
-        (feature) => feature.type === "PersistentCertifier"
-      )?.certifierId;
-
-      if (!certifierId) {
-        throw tenantIsNotACertifier(organizationId);
-      }
+      const retrieveCertifiedId = getTenantCertifierId(requesterTenant.data);
```

Comment:
My comment was about renaming the function, not the variable, rename it as `retrieveCertifierId`